addons/binding/org.openhab.binding.gpio/README.md

```diff
@@ -0,0 +1,31 @@
+# GPIO
```

Comment:
This readme looks limited can you please add examples how to configure things (also via the things / items and sitemap config)